docs/proposals/04-new-core-gardener-cloud-apis.md

```diff
@@ -0,0 +1,698 @@
+  #
+  # If you use your own domain (not the default domain of your landscape) then you have to add additional keys to this secret.
+  # The reason is that the DNS management is not part of the Gardener core code base but externalized, hence, it might use other
+  # key names than Gardener itself.
+  # The actual values here depend on the DNS extension that is installed to your landscape.
+  # For example, check out https://github.com/gardener/external-dns-management and find a lot of example secret manifests here:
+  # https://github.com/gardener/external-dns-management/tree/master/examples
+
+---
+apiVersion: core.gardener.cloud/v1alpha1
+kind: SecretBinding
+metadata:
+  name: secretbinding1
+  namespace: garden-core
+secretRef:
+  name: secret1
+# namespace: namespace-other-than-'garden-core' // optional
+quotas: []
+# - name: quota-1
+# # namespace: namespace-other-than-'garden-core' // optional
+```
+
+### `Quota` resource
+
+Special note: No modifications needed compared to the current `garden.sapcloud.io/v1beta1.Quota` resource.
+
+```yaml
+apiVersion: core.gardener.cloud/v1alpha1
+kind: Quota
+metadata:
+  name: trial-quota
+  namespace: garden-trial
+spec:
+  scope: secret
+# clusterLifetimeDays: 14
+  metrics:
+    cpu: "200"
+    gpu: "20"
+    memory: 4000Gi
+    storage.standard: 8000Gi
+    storage.premium: 2000Gi
+    loadbalancer: "100"
+```
+
+### `BackupBucket` resource
+
+Special note: This new resource is cluster-scoped.
+
+```yaml
+# See also: https://github.com/gardener/gardener/blob/master/docs/proposals/02-backupinfra.md.
+
+apiVersion: v1
+kind: Secret
+metadata:
+  name: backup-operator-provider
+  namespace: backup-garden
+type: Opaque
+data:
+  # <some-provider-specific data keys>
+  # https://github.com/gardener/gardener-extensions/blob/master/controllers/provider-alicloud/example/30-backupbucket.yaml#L9-L11
+  # https://github.com/gardener/gardener-extensions/blob/master/controllers/provider-aws/example/30-backupbucket.yaml#L9-L10
+  # https://github.com/gardener/gardener-extensions/blob/master/controllers/provider-azure/example/30-backupbucket.yaml#L9-L10
+  # https://github.com/gardener/gardener-extensions/blob/master/controllers/provider-gcp/example/30-backupbucket.yaml#L9
+  # https://github.com/gardener/gardener-extensions/blob/master/controllers/provider-openstack/example/30-backupbucket.yaml#L9-L13
+
+---
+apiVersion: core.gardener.cloud/v1alpha1
+kind: BackupBucket
+metadata:
+  name: providerprofile1-random[:5]
+  ownerReferences:
+  - kind: ProviderProfile
+    name: providerprofile1
+spec:
+  provider:
+    type: <some-provider-name> # {aws,azure,gcp,...}
+    region: europe-central-1
+  secretRef:
+    name: backup-operator-provider
+    namespace: backup-garden
+status:
+  lastOperation:
+    description: Backup bucket has been successfully reconciled.
+    lastUpdateTime: '2020-04-13T14:34:27Z'
+    progress: 100
+    state: Succeeded
+    type: Reconcile
+  observedGeneration: 1
+  seed: seed1
```

Comment:
Move this to spec. Since we have decided to  use scheduler for seed which requires this to be part of spec.

Reply:
OK, makes sense, will adapt it. Thanks.

Reply:
Did you forgot to push the change here?

Reply:
Isn't https://github.com/gardener/gardener/pull/1284/commits/4524895d9c58ccf9860119b4edd09dad701aa4ad#diff-bf2774d9954baab517306db45a5b80bbR428 what you meant?

Reply:
Nope, i mentioned for `BackupBucket` resource? Its good that you changed it for backupEntry resource (though we are not introducing scheduler for backupEntry immediately.). But the change needs to be done in "BackupBucket" resource spec as well.

Reply:
Done now, thanks!